docs/sdks/uid2-sdk-ref-python.md

```diff
@@ -76,6 +76,68 @@ Available information returned through the SDK is outlined in the following tabl
 | `KeysNotSynced` | The client has failed to synchronize keys from the UID2 service. |
 | `VersionNotSupported` |  The client library does not support the version of the encrypted token. |
 
+
+
+## Usage for UID2 Sharers
+
+A UID2 sharer is any participant that wants to share UID2s with another participant. Raw UID2s must be encrypted into UID2 tokens before sending them to another participant. For an example of usage, see [com.uid2.client.test.IntegrationExamples](https://github.com/IABTechLab/uid2-client-java/blob/master/src/test/java/com/uid2/client/test/IntegrationExamples.java) (`runSharingExample` method).
+
+The following instructions provide an example of how you can implement sharing using the UID2 SDK for Java, either as a sender or a receiver.
+
+1. Create an IUID2Client reference:
+ 
+   ```java
+   from uid2_client import Uid2Client
+   client = Uid2Client(base_url, auth_key, secret_key)
+   ```
+2. Refresh once at startup, and then periodically (for example, every hour):
+
+   `keys = client.refresh_keys()`
+
+3. Senders: 
```

Comment:
Python is a bit different comparing to java/c++/c# sdk (can get error status in the result of the encrypt/decrypt functions) and the only way to detect encryption/decryption error is to catch error during the encrypt/decrypt operation.

i think for Senders we can just keep "2." but instead of
_If encryption succeeded, send the UID2 token to the receiver:_

reword to:
_Call the `encrypt ` function and if encryption succeeded, send the UID2 token to the receiver:_

Reply:
Done.